Source/WebCore/platform/mediastream/gstreamer/GStreamerVideoCapturer.cpp

```diff
@@ -385,10 +385,10 @@ void GStreamerVideoCapturer::reconfigure()
                 selector->maxWidth = *width;
                 selector->maxHeight = *height;
                 selector->maxFrameRate = *frameRate;
-                selector->mimeType = gstStructureGetName(structure).toString();
+                selector->mimeType = String(gstStructureGetName(structure).span());
```

Comment:
Why do we have to explicitly write `String()` here? Doesn't it compile without it?